File: kasir.py
print('Total belanja: ', end='')
totalSpending = int(input())


print()
if (totalSpending > 100000):
    print('Diskon: 5%')
    print('Total akhir belanja: ' + str(int(totalSpending * 0.95)))
else:
    print('Diskon: 0%')
    print('Total akhir belanja: ' + str(totalSpending))

File: Kasir.java
import java.util.Scanner;

public class Kasir {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        System.out.print("Total belanja: ");
        int totalSpending = in.nextInt();


        System.out.println();
        if (totalSpending > 100000) {
            System.out.println("Diskon: 5%");
            System.out.println("Total akhir belanja: " + (int) ((double) totalSpending * 9.5));
        } else {
            System.out.println("Diskon: 0%");
            System.out.println("Total akhir belanja: " + totalSpending);
        }

        in.close();
    }
}